### ISSUE TITLE
Just installed on El Capitan Server, but still showing old built in version after restart

### ISSUE THREAD

**Issue by ssuess** [liip/php-osx#160]
Hi, I just installed (or reinstalled I guess, php-osx was installed a few months back but stopped working after someone updated the system). The install went off without a hitch, no errors. but when I loaded phpinfo, it was still the built in version (5.5.27). So I then placed the following line in /private/etc/apache2/httpd.conf (because it was NOT there):

LoadModule php5_module /usr/local/php5/libphp5.so

And restarted, but still shows old version. Anyone have a clue why this isn't working? I checked permissions on the file, which seems ok, and I am at a loss at the moment....

**Comment by ssuess** [comment]
After finding some other info on the internet (specific to Server), I also tried to add the LoadModule directive in 

/Library/Server/Web/Config/apache2/httpd_server_app.conf

That made a difference. It broke everything. PHP would not work at all.

**Comment by ssuess** [comment]
Tried again and found this in the system log:
 servermgr_web[526]: XSWebConfig:ERROR: 
httpd: Syntax error on line 181 of /Library/Server/Web/Config/apache2/httpd_server_app.conf: Cannot load /usr/local/php5/libphp5.so into server: dlopen(/usr/local/php5/libphp5.so, 10): Symbol not found: _environ\n  Referenced from: /usr/local/php5/libphp5.so\n  Expected in: /usr/sbin/httpd\n in /usr/local/php5/libphp5.so

**Comment by ssuess** [comment]
Maybe this has to do with El Capitan's new root protection?

**Comment by ssuess** [comment]
well, thinking maybe I needed to delete and reinstall everything, I made things much worse. removed everything in the registry and then ran an install and got these errors that I can't get past:

$ curl -s http://php-osx.liip.ch/install.sh | bash -s 5.6
****
[WARNING]
Detected OS X El Capitan 10.11. As this is quite new, there may be issues still. Your mileage may vary.
****
Get packager.tgz
Unpack packager.tgz
Please type in your password, as we want to install this into /usr/local
Start packager (may take some time)
Traceback (most recent call last):
  File "/usr/local/packager/packager.py", line 553, in <module>
    c.run()
  File "/usr/local/packager/packager.py", line 41, in run
    cmd_obj.run(self, params, options)
  File "/usr/local/packager/packager.py", line 144, in run
    registry = PackageRegistry()
  File "/usr/local/packager/packager.py", line 430, in __init__
    self.packages = self.__parseRegistry(self.__registry)
  File "/usr/local/packager/packager.py", line 442, in __parseRegistry
    package, version = line.split(':')
ValueError: need more than 1 value to unpack

I give up for now.

**Comment by schnoddelbotz** [comment]
Server.app comes with a GUI to "manage" apache config; It makes manual editing of httpd.conf and friends more or less impossible (configs will be re-written from templates each time you tweak a setting through GUI). I'd recommend stating that these packages are not intended to run on Macs with Server.app installed.
See also: [Where is the server app changing the Apache configuration?](http://serverfault.com/questions/446132/os-x-server-how-where-is-the-server-app-changing-the-apache-configuration)

**Comment by ssuess** [comment]
Just revisiting this after many months and finally success! I had to completely REMOVE the packager folder and then just ran the normal command and it replaced/rebuilt it. I ALSO had to follow the VERY helpful server instructions [here](https://github.com/liip/php-osx/issues/174). But once I did that, everything is purring along nicely on Mac OS Sierra Server. Oh, one other very important thing: I had to disable xdebug to reap performance gains over the built in 5.6. Hope this helps someone else.
